demo-shell/src/app/components/process-service/process-service.component.scss

```diff
@@ -11,6 +11,10 @@
             margin: 4px;
             box-shadow: 0 2px 2px 0 rgba(0,0,0,.14), 0 3px 1px -2px rgba(0,0,0,.2), 0 1px 5px 0 rgba(0,0,0,.12);
             padding: 10px;
+
+            .mat-expansion-panel-header {
```

Comment:
The issue is related to the material list space and not on the material expansion panel.
In addition, the fix must to be done on the ADF filter component and not on the demo shell side.
<img width="1174" alt="" src="https://user-images.githubusercontent.com/14923305/39864588-01e7c68e-5442-11e8-95fb-ce9ffa96595c.png">